editor/src/clj/editor/properties_view.clj

```diff
@@ -235,21 +236,25 @@
                      (.isControlDown event) (* 0.1)
                      (neg? max-delta) -)]
     (when (> (abs max-delta) 1)
-      (g/transact
-        (for [node-id node-ids]
-          (let [current-value (cond-> (g/node-value node-id key) to-type to-type)
-                new-value (cond-> (drag-update-fn current-value update-val) 
-                            from-type from-type
-                            (:min edit-type) (max (:min edit-type))
-                            (:max edit-type) (min (:max edit-type)))]
-            (concat (g/operation-sequence (ui/user-data target ::op-seq))
-                    (g/set-property node-id key new-value)))))
-      (ui/user-data! target ::position [x y])
-      (when (apply = (properties/values property))
-        (update-ui-fn [(cond-> (g/node-value (first node-ids) key)
-                         to-type to-type)]
-                      (properties/validation-message property)
-                      (properties/read-only? property))))))
+      (let [op-seq (ui/user-data target ::op-seq)
+            values (properties/values property)
+            set-operations (properties/resolve-set-operations property values)]
+        (g/transact
+          (for [[node-id prop-kw] set-operations
+                :let [current-value (cond-> (g/node-value node-id prop-kw) to-type to-type)
+                      new-value (cond-> (drag-update-fn current-value update-val)
+                                  from-type from-type
+                                  (:min edit-type) (max (:min edit-type))
+                                  (:max edit-type) (min (:max edit-type)))]]
+            (concat (g/operation-sequence op-seq)
+                    (g/set-property node-id prop-kw new-value))))
+        (ui/user-data! target ::position [x y])
+        (when (apply = values)
+          (let [[node-id prop-kw] (first set-operations)]
```

Comment:
Hmm, now that I think of it, this code might be unsafe since prop-kw is not guaranteed to exist. Here is an example of such a property definition without `:prop-kw`: https://github.com/defold/defold/blob/690c5d630d7dd3e5b0da935a6f5c1bc941fef17d/editor/src/clj/editor/sprite.clj#L430-L437

There is another place above that uses `g/node-value` on `prop-kw`: https://github.com/defold/defold/blob/8855500c663f4c891fedbe122d0ecf40dbf78ac1/editor/src/clj/editor/properties_view.clj#L244